test/hmactest.c

```diff
@@ -81,6 +85,62 @@ static struct test_st {
 };
 # endif
 
+# ifndef OPENSSL_NO_SM3
+/* Example vectors below taken from
+ * GM/T 0042-2015 Appendix D.3
+ */
+struct testvec_st {
+    const char key[64];
+    int ksize;
+    const unsigned char plaintext[128];
+    int psize;
+    const char* digest;
+} hmac_sm3_tv[] = {
+    {
+        .key    = "\x01\x02\x03\x04\x05\x06\x07\x08"
```

Comment:
Our coding style doesn't allow named initialisers.

Reply:
I will fix it